virt-v2v/cold/entrypoint.go

```diff
@@ -114,7 +116,14 @@ func buildCommand() []string {
 		fmt.Println("Error creating directory  ", err)
 		os.Exit(1)
 	}
-	virtV2vArgs = append(virtV2vArgs, "-o", "local", "-os", DIR)
+	virtV2vArgs = append(virtV2vArgs, "-o", "kubevirt")
+
+	if checkEnvVariablesSet("V2V_NewName") {
+		virtV2vArgs = append(virtV2vArgs, "-on", os.Getenv("V2V_NewName"))
+		nameChanged = true
+	}
+
+	virtV2vArgs = append(virtV2vArgs, "-os", DIR)
```

Comment:
i find the split of these args from above unnecessary, but i'm not strongly against it.

Reply:
the order meters in this case IIRC, so it was divided due to that.

Reply:
Order doesn't matter (except obviously things like `-os` needs to have the directory parameter after).  On the other hand, having the order `-i ... -o ...` does make it easier for humans to read.